soperator/modules/slurm/variables.tf

```diff
@@ -248,6 +248,12 @@ variable "accounting_enabled" {
   default     = false
 }
 
+variable "protected_secret" {
```

Comment:
Let's rename it to something like `use_protected_secret` or `protect_secret`. Current name is misleading as it looks like some secret is expected rather than flag to protect it or not